app/src/main/java/com/poupa/vinylmusicplayer/preferences/ExportSettingsPreferenceDialog.java

```diff
@@ -73,7 +72,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     }
 
     private static String getCurrentFormattedDateTime() {
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy_MM_dd_HH_mm_ss", Locale.getDefault());
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyyMMddHHmmssZ", Locale.getDefault());
```

Comment:
"yyyyMMddTHHmmssZ" please (missing T between date and time part)

Reply:
Unfortunatly I get `java.lang.IllegalArgumentException: Illegal pattern character 'T'` Exception when I add the 'T'.

Reply:
Ah ok.